notifications/usernotificationreporttypes.go

```diff
@@ -86,8 +86,9 @@ type AlertConfig struct {
 }
 
 type AlertScope struct {
-	Cluster    string   `json:"cluster,omitempty" bson:"cluster,omitempty"`
-	Namespaces []string `json:"namespaces,omitempty" bson:"namespaces,omitempty"`
+	Cluster          string   `json:"cluster,omitempty" bson:"cluster,omitempty"`
+	ClusterShortName string   `json:"clusterShortName,omitempty" bson:"clusterShortName,omitempty"`
```

Comment:
why?
Cluster short name is not reliable data - it is only for user display

Reply:
If needed than add the cluster guid and fetch the short name from the cluster in config service